synapse/lib/ast.py

```
@@ -459,7 +479,12 @@ def getRuntVars(self, runt):
         # no nodes and a runt safe value should execute once
         if node is None and self.kids[1].isRuntSafe(runt):
 
-            async for item in s_coro.agen(await self.kids[1].runtval(runt)):
+            # TODO: remove when storm is all objects
+            valu = await self.kids[1].compute(runt)
+            if isinstance(valu, dict):
+                valu = valu.items()
```

this is dangerous since you can raise an exception inside of iteration